libconfig submodule forgotten into minimum checkout

Hi, 
while attempting to build Lima following the doc steps (http://lima.blissgarden.org/install_linux.html), I realized that libconfig submodule is not included into "Commands for a minimum checkout"

Following strictly these instructions and executing "make" will lead to the following error :

```
make[2]: quittant le répertoire « /home/tangosrv/SW_tango/Maquette/camera/src/Lima-non-recursif/third-party/Sps/Src »
cd libconfig && ./configure --prefix=/home/tangosrv/SW_tango/Maquette/camera/src/Lima-non-recursif/install/Lima/
/bin/sh: ./configure: not found
make[1]: *** [libconfig.confsrc] Erreur 127
make[1]: quittant le répertoire « /home/tangosrv/SW_tango/Maquette/camera/src/Lima-non-recursif/third-party »
make: *** [third-party.src] Erreur 2
```

To avoid this, one should modify 
`git submodule init third-party/Processlib third-party/Sps`
into the following:
`git submodule init third-party/Processlib third-party/Sps third-party/libconfig`

--- a/source/install_linux.rst
+++ b/source/install_linux.rst
@@ -23,13 +23,13 @@ As Lima is not packaged,the only way for now is to retreived it from the git rep
 
   seb@pcbliss01:~/$ git clone --recursive git://github.com/esrf-bliss/Lima.git
 
-**Commands for a minimum chekout to get all source needed for a particular camera:**
+**Commands for a minimum checkout to get all source needed for a particular camera:**
 
 .. code-block:: bash
 
   seb@pcbliss01:~/$ git clone git://github.com/esrf-bliss/Lima.git
   seb@pcbliss01:~/$ cd Lima
-  seb@pcbliss01:~/Lima$ git submodule init third-party/Processlib third-party/Sps
+  seb@pcbliss01:~/Lima$ git submodule init third-party/Processlib third-party/Sps third-party/libconfig
   seb@pcbliss01:~/Lima$ git submodule init camera/CAMERA_YOU_WANT_TO_COMPILE
   seb@pcbliss01:~/Lima$ git submodule update
 

